scroll-animations-1/Overview.bs

```diff
@@ -465,40 +465,6 @@ not by wall-clock time, but by the progress of scrolling in a [=scroll container
     :   <<percentage>>
     ::  A percentage distance along {{scrollSource}}'s scroll range in {{orientation}}.
 
-:   <dfn attribute for=ScrollTimeline>timeRange</dfn>
```

Comment:
Another option would be keeping timeRange and just having auto map to the default 100s value. In practice I expect most people to use the automatic duration / timeRange though and so would suggest that we keep this as an option for later if needed.

Reply:
I rather remove the value for now.

Reply:
Agreed, I think it's easier to add the property back if it proves to still be necessary than to have a property which we may have difficulty removing if it becomes unergonomic - i.e. if we decide that percentages have different treatment and ScrollTimeline should only output percentages.